cms/static/js/certificates/views/signatory_details.js

```diff
@@ -73,6 +73,7 @@ function ($, _, str, Backbone, gettext, TemplateUtils, ViewUtils, BaseView, Sign
                     actionableModel.save({}, {
                         success: function() {
                             actionableModel.setOriginalAttributes();
+                            self.model.setOriginalAttributes();
```

Comment:
When user clicks on signatory window in order to edit then we are going to update the particular signatory instead of saving the whole certificate object that's why we need to update the original state of a particular signatory in success callback.